about.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="stylesheet" href="./main.css">
+    <link rel="stylesheet" href="./about.css">
+    <link rel="shortcut icon" href="dodgicon.png" type="image/x-icon">
+    <title>About- The 2021 Design & Content Conference</title>
+</head>
+
+<body>
+    <header>
+        <nav>
+            <a class="logo" href="./index.html">Design & Content</a>
+            <img class="hamb-icon" src="menuicon.png" alt="menu icon">
+            <ul class="menu-links">
+                <li class="menu-list close"><img class="hamb-close-btn" src="closeicon.png" alt="close button"></li>
+                <li class="menu-list"><a class="menu-link" href="./about.html">About</a></li>
+                <li class="menu-list"><a class="menu-link" href="./index.html#program">Program</a></li>
+                <li class="menu-list"><a class="menu-link" href="./about.html">Join</a></li>
+                <li class="menu-list"><a class="menu-link" href="./about.html#main-about-me">Sponsor</a></li>
+                <li class="menu-list"><a class="menu-link" href="./about.html#events">News</a></li>
+            </ul>
```

Comment:
- [ ] Good job implementing this mobile menu. You should consider adding a link back to the home page, you can either add the logo or a simple "HOME" link.

![image](https://user-images.githubusercontent.com/66967274/142280318-b5817116-15bb-4938-bc08-696264e631c8.png)